plugin/illuminate.vim

```diff
@@ -31,10 +31,12 @@ endif
 " Commands {{{
 command! -nargs=0 IlluminationDisable call illuminate#disable_illumination()
 command! -nargs=0 IlluminationEnable call illuminate#enable_illumination()
+command! -nargs=0 IlluminationToggle call illuminate#toggle_illumination()
 
 " Keep these for backwards compatibility
 command! -nargs=0 DisableIllumination :IlluminationDisable
 command! -nargs=0 EnableIllumination :IlluminationEnable
+command! -nargs=0 ToggleIllumination :IlluminationToggle
```

Comment:
This isn't needed. I only left the first two to maintain backwards compatibility incase someone has a mapping for it already. Illuminate<option> is the naming style im going with for <tab> completion.

Reply:
Ok, I removed this, sorry for the delay